rules/docker-restapi.mk

```diff
@@ -19,4 +19,5 @@ endif
 $(DOCKER_RESTAPI)_CONTAINER_NAME = restapi
 $(DOCKER_RESTAPI)_RUN_OPT += --cap-add NET_ADMIN --privileged -t
 $(DOCKER_RESTAPI)_RUN_OPT += -v /var/run/redis/redis.sock:/var/run/redis/redis.sock
-$(DOCKER_RESTAPI)_RUN_OPT += -p=8090:8090/tcp
+$(DOCKER_RESTAPI)_RUN_OPT += -v /etc/sonic/certificates:/etc/sonic/certificates:ro
+$(DOCKER_RESTAPI)_RUN_OPT += -p=8081:8081/tcp
```

Comment:
why change port number?

Reply:
8081 is the https endpoint.
8090 is for http endpoint

---
In reply to: [404614708](https://github.com/Azure/sonic-buildimage/pull/4383#discussion_r404614708) [](ancestors = 404614708)

Reply:
should we then bind both?

Reply:
Yes, i think we should bind both

Reply:
If we don't want to bring-up the http endpoint at all, why do we need to bind the 8090 port?

Reply:
it might be required for testing, we can probably remove it later

Reply:
We test the restapi server using the restapi test container image generated by https://github.com/Azure/sonic-restapi/blob/master/build.sh and the configuration file https://github.com/Azure/sonic-restapi/blob/master/supervisor/rest_api_test.conf which will enable the http endpoint. Therefore I don't think we need to keep the port open here.

Reply:
I have added port 8090 also now

---
In reply to: [406905231](https://github.com/Azure/sonic-buildimage/pull/4383#discussion_r406905231) [](ancestors = 406905231)